astropy/io/fits/fitstime.py

```diff
@@ -355,6 +355,10 @@ def time_to_fits(table):
     location = None
 
     for col in time_cols:
+        if col.info.serialize_method['fits'] == 'formatted_value':
```

Comment:
Could you add a code comment here, along the lines of: "by default, we write time objects in full precision, i.e., we store both jd1 and jd2 (serialize_method['fits'] = 'jd1_jd2'), unless the user explicitly chose to store the formatted value." (not great text, do adjust!)